src/test/java/com/google/cloud/spanner/hibernate/util/TestEntity.java

```diff
@@ -31,7 +31,7 @@
  * @author Chengyuan Zhao
  */
 @Entity
-@Table(name = "test_table")
+@Table(name = "`test_table`")
```

Comment:
Is this required in the annotation? It looks a bit ugly this way an unconventional.

Reply:
That's not required, but we need to support that. There are some tests in hibernate that use it.